python/lib/dependabot/python/update_checker/pipenv_version_resolver.rb

```diff
@@ -62,11 +64,7 @@ def resolvable?(version:)
           @resolvable ||= {}
           return @resolvable[version] if @resolvable.key?(version)
 
-          @resolvable[version] = if fetch_latest_resolvable_version_string(requirement: "==#{version}")
-                                   true
-                                 else
-                                   false
-                                 end
+          @resolvable[version] = !!fetch_latest_resolvable_version_string(requirement: "==#{version}")
```

Comment:
This was perhaps an unnecessary change as it's purely cosmetic but it bothered me 🙈